runtime/src/accounts_db.rs

```diff
@@ -4230,6 +4230,23 @@ impl AccountsDb {
         result
     }
 
+    fn sort_storages(storages: &[SnapshotStorage]) -> Vec<&Vec<Arc<AccountStorageEntry>>> {
```

Comment:
Lately I've been seeing Carl change function parameter types from slices/containers to iterators. [Here's an example](https://github.com/solana-labs/solana/pull/17319/files#diff-1090394420d51617f3233275c2b65ed706b35b53b115fe65f82c682af8134a6fR1504). I dunno if that is either more performant and/or more idiomatic, but figured I'd point that out here for `storages`.